staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -63,6 +63,7 @@ type AuditLogOptions struct {
 	MaxAge     int
 	MaxBackups int
 	MaxSize    int
+	Format     string
```

Comment:
Yeap, it will get such error:
cannot use &o.Format (type *format) as type *string in argument to fs.StringVar
cannot use o.Format (type format) as type string in argument to fs.StringVar